skbio/alignment/tests/test_tabular_msa.py

```diff
@@ -874,5 +874,45 @@ def test_from_dict_to_dict_roundtrip(self):
         self.assertIs(d1[42], d2[42])
 
 
+class TestIterPositions(unittest.TestCase):
+    def setUp(self):
+        self.msa_empty = TabularMSA(())
+        self.msa = TabularMSA((DNA('ACGT'), DNA('TGCA')))
+        self.expected = [[Sequence('A'), Sequence('T')],
```

Comment:
@ebolyen I reviewed the discussion and everything makes sense. I also fall under the camp of "this feels weird but I like it". One question from [your previous comment](https://github.com/biocore/scikit-bio/pull/1128#discussion_r39421819):

Is concatenating the metadata in some way going to be necessary functionality?